spot-client/src/spot-tv/ui/components/setup/select-media/select-media.js

```diff
@@ -181,16 +189,16 @@ class SelectMedia extends React.Component {
                 </div>
                 <div className = 'setup-buttons'>
                     <Button
-                        disabled = { this.state.saving }
+                        disabled = { saving }
                         onClick = { this._onSubmit }
                         qaId = 'device-selection-submit'>
-                        Next
+                        { t('buttons.submit') }
```

Comment:
changed from next to submit?

Reply:
I liked submit better, especially for standalone device selection that isn't part of the setup flow. Change it back?